docs/src/main/sphinx/object-storage/file-system-azure.md

```diff
@@ -91,3 +91,14 @@ Storage:
 * - `azure.oauth.secret`
   - A OAuth 2.0 client secret for the service principal.
 :::
+
+## Access multiple storage accounts
+
+To allow Trino to access multiple Azure storage accounts from a single
+catalog configuration, you can use [](azure-oauth-authentication) with a
+service principal. First, create a service principal in Azure Active
+Directory (App registrations) and save the client secret. Then, grant the
+service principal access by assigning appropriate roles, such as Storage
```

Comment:
I think Storage Blob Data Contributor is an actual name of a role .. so please bold it